### DISCUSSION TITLE
Root Escalation [Unenrollment Exploit]

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#349] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Discord link to the kajig https://discord.com/channels/419123358698045453/1052240595571179540

Join the tn discord for it discord.gg/unblock

### Preview

https://coolelectronics.me/unroll
thread for unroll and discussions related to unroll

You can thank phene for the deletion of root escalation

**Comment by Aka-but-nice** [comment]
So basically the steps are unenroll your chromebook,  how about you refer to #348  or #347 if you don't want to get into legal trouble

**Reply by rwg100** [reply]
breaking the law is fun

**Reply by 3kh0** [reply]
My school does not care about that

**Reply by Aka-but-nice** [reply]
@3kh0 Your school must be in ohio💀

**Reply by 3kh0** [reply]
lol

**Reply by ineedyoutubeworkingonschoolcromebook** [reply]
3kho can u make a java script to make youtube working bc when ever i open youtube and click on a non school related video it says please reconnect when i have full bars of wifi

**Comment by Alex2Mayhem** [comment]
> Discord link to the kajig https://discord.com/channels/419123358698045453/1052240595571179540
> 
> Join the tn discord for it discord.gg/unblock
> 
> ### Preview
> https://coolelectronics.me/unroll thread for unroll and discussions related to unroll
> 
> You can thank phene for the deletion of root escalation

Yeah i'm gonna try this. NOT. Legit living in ohio? #187

**Reply by Aka-but-nice** [reply]
Fax

**Comment by rwg100** [comment]
*clicks i am a useless memeber of society*
*instantly regrets it*

**Reply by Calamitou** [reply]
lol rickrolled

**Comment by ineedyoutubeworkingonschoolcromebook** [comment]
3kho can u make a java script to make youtube working bc when ever i open youtube and click on a non school related video it says please reconnect when i have full bars of wifi

**Reply by Aka-but-nice** [reply]
Man 3kh0 can't code, you can ask him though, he seems to have, mostly knowledge of python, but he seems pretty good at Javascript.
Also respeak your problem as I didn't understand.

**Reply by amelia-bartlett** [reply]
> 3kho can u make a java script to make youtube working bc when ever i open youtube and click on a non school related video it says please reconnect when i have full bars of wifi

you need to at least give us a screenshot of whatever tf this is
i bet you its a network level block
